### ISSUE TITLE
Paging tab completion

### ISSUE THREAD

**Issue by joelphillips** [ocaml-community/utop#29]
I'd like a way to page through the possible completions.  It seems like the most natural way would be for subsequent presses of tab to advance to the next page.

**Comment by sanette** [comment]
> I'd like a way to page through the possible completions. It seems like the most natural way would be for subsequent presses of tab to advance to the next page.
 
I fully agree. Having to use 4 keys (ALT, LEFT, RIGHT, DOWN) to select a completion is very cumbersome, especially when one is used to do everything with repeated TAB presses.
